pyiron/atomistics/structure/atoms.py

```diff
@@ -693,7 +694,9 @@ def center(self, vacuum=None, axis=(0, 1, 2)):
         translation = np.zeros(3)
         for i in axes:
             nowlen = np.sqrt(np.dot(c[i], c[i]))
-            self.cell[i] *= 1 + longer[i] / nowlen
+            cell = self.cell
```

Comment:
or `cell = self.cell.copy()` as cell is an numpy array you still have a link when you use `cell = self.cell`